src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2931,4 +2932,32 @@ BOOST_AUTO_TEST_CASE(ConvolutionLayerPaddingTest)
   module2.Backward(std::move(input), std::move(output), std::move(delta));
 }
 
+/**
+ * Simple Separable Convolution layer test.
+ */
+BOOST_AUTO_TEST_CASE(SimpleSeparableConvolutionLayerTest)
+{
+  arma::mat output, input, delta;
+  SeparableConvolution<> module1(1, 4, 2, 2, 1, 1, 0, 0, 3, 3, 1, "valid");
```

Comment:
Hi @zoq, I needed your help while implementing tests for this layer.
I ran the following test in [python](https://colab.research.google.com/drive/1VSHDd7Tsg5TrCwR4vZXZ361OjYGOSWGE#scrollTo=zoQfUhjep2cg) and got sum as 111 but when I run the same test using Convolution Layer I get 113. Could you help me in figuring out what I did wrong.
Thanks.